nova_plugin/tests/resources/test-networks-relationships-blueprint.yaml

```diff
@@ -0,0 +1,82 @@
+
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - plugin.yaml
```

Comment:
exactly, it seemed to me like this shouldn't work.
should probably indeed make this more robust somewhat